src/docs/docs/api.mdx

```diff
@@ -0,0 +1,102 @@
+
+### Public API
+
+Our public API is documented at [https://docs.sentry.io/api/](https://docs.sentry.io/api/).
+
+### Private Attributes in a Public Endpoint
+
+You can have private attributes within a public endpoint.
+
+As an example: [https://docs.sentry.io/api/teams/retrieve-a-team/](https://docs.sentry.io/api/teams/retrieve-a-team/). The response has multiple attributes:
+
+```json
+{
+  "id": "2",
+  "slug": "the-interstellar-jurisdiction"
+  ...
+}
+```
+
+Let's say we also return a `nickname` for a team. If what is returned is not documented as a response in our public documentation, that attribute is a private attribute within a public endpoint.
+
+## API versioning
+
+Currently, Sentry's API is v0 and is considered to be in draft phase. While we don't expect our public endpoints to change greatly, keep in mind that our API is still under development.
+
+## Making an Endpoint Public
+
+When you make an endpoint public, you're committing to always returning at least the set of defined attributes when that endpoint is called. As a result, the attribute(s) **cannot** **be removed** from an endpoint after they are public. Additionally, the type of the attribute **cannot** **be changed** once the attribute is public. The reason the attribute and its type cannot be changed is because both in the wild and managed clients are relying on the public attribute to return what is expected from that endpoint.
+
+You **can** add attributes to a public endpoint.
+
+## When to make an API public?
+
+Making an API public offers considerable benefits. A public API allows our customers to incorporate Sentry into their workflows. It also gives Sentry more visibility in our customers' organizations ("Woah that's cool data, where'd it come from?"..."Sentry!"). Remember, however, making an API public means that once an endpoint is public, you can only add to it: You cannot make *any* breaking changes.
+
+As a guide, use these questions:
+
+1. Is the feature for which you're making the endpoint stable?
+2. Will the API change substantially in the future?
+3. Does the API offer value to customers who are accessing it?
+
+If your answers are Yes, No and Yes, you're in business - make the API public.
+
+## How to make an endpoint public?
+
+We use [Open API Spec 3](https://swagger.io/docs/specification/about/) to write our API documentation. You can find that content here: [https://github.com/getsentry/sentry/tree/master/api-docs.](https://github.com/getsentry/sentry/tree/master/api-docs) You can find the tests here: [https://github.com/getsentry/sentry/tree/master/tests/apidocs](https://github.com/getsentry/sentry/tree/master/tests/apidocs).
+
+Once you make changes to an endpoint and merge the change into Sentry, a Github Action will update the schema in [sentry-api-schema](https://github.com/getsentry/sentry-api-schema) with a build version. To make your changes go live:
+
+1. Get the latest SHA from sentry-api-schema.
```

Comment:
you could link to https://github.com/getsentry/sentry-api-schema/commits/main to make it easier